pulsar/log/wrapper_logrus.go

```diff
@@ -0,0 +1,134 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package log
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+// logrusWrapper implements Logger interface
+// based on underlying logrus.FieldLogger
+type logrusWrapper struct {
+	l logrus.FieldLogger
+}
```

Comment:
If interface Logger's method is a sub set of `*logrus.Logger`, you can directly new a `logrus.Logger` as interface `Logger` to be the default logger without define this wrapper.

Reply:
thanks for your suggestion :-). Actually, the method set of interfaces is not a subset of logrus.Logger
*  the logger/entry interfaces defined in pulsar-client-go are independent of any existing logger implementations.
* the logger has its own `Fields` type even though it's similar with `logrus`, but they are not the same.